packages/create-itp-react-app/templates/app/src/__mockdata__/state.ts

```diff
@@ -0,0 +1,27 @@
+import { features } from './features';
+
+const entitiesState = (collection: any) => collection.reduce(
+  (state: any, object: any) => ({
+    ...state,
+    [object.id]: object,
+  }),
+  {},
+);
+
+const uiState = (collection: any = []) => ({
```

Comment:
Not sure how this can be any if default is array and map is called on it

Reply:
added extra typing: [b0b54b0](https://github.com/inthepocket/itp-react-components/pull/42/commits/b0b54b015ad534f6603460554a15263bcab52df9)